src/gcl/MeshVelocityEdgeAlg.C

```diff
@@ -71,8 +71,7 @@ MeshVelocityEdgeAlg<AlgTraits>::MeshVelocityEdgeAlg(
     19, &isoParCoords_[0], isoCoordsShapeFcnHostView_.data());
   Kokkos::deep_copy(isoCoordsShapeFcnDeviceView_, isoCoordsShapeFcnHostView_);
 
-  auto scsFaceNodeMapHostView =
-    Kokkos::create_mirror(scsFaceNodeMapDeviceView_);
+  Kokkos::View<int[12][4], Kokkos::HostSpace> scsFaceNodeMapHostView;
```

Comment:
this does not initialize the view, the pointer will be a null ptr. Needs a string as argument to the constructor.

Reply:
so a `Kokkos::View<int[12][4], Kokkos::HostSpace> scsFaceNodeMapHostView (<some string>)` ?

Reply:
yeah, but that will not allow for the deep_copy below because device and host view will have different layouts.